README.md

```diff
@@ -1,23 +1,16 @@
 # SumoCoders FrameworkCoreBundle
+This bundle is created and maintained by [SumoCoders](https://github.com/sumocoders). It contains a set of basic tools that enable us to build an application in a shorter timespan. The bundle is intended to be used together with our [npm package](https://github.com/sumocoders/FrameworkStylePackage).
 
-The FrameworkCoreBundle will add some functionality which will enable you build
-application in a short timespan.
+## Installation
+```
+$ composer require sumocoders/framework-core-bundle
```

Comment:
zou er hier niet beter uitleg staan van create skeleton via composer?

Reply:
Aangepast.

Weet trouwens niet of major version nodig zal zijn. Voorlopig lijkt alles backwards compatible correct.

Reply:
Aja, de namespace is niet veranderd zeker?

Reply:
Nope, voor de projecten die de bundle inladen zou er niets mogen veranderen.